src/Components/Banner.js

```diff
@@ -1,5 +1,36 @@
 import React from "react";
+import styles from "../Css/Banner.module.css";
 
-export default function Banner() {
-  return <></>;
+export default function App() {
+  return (
+    <div className={`${styles.App}`}>
+      <div className={`${styles.main}`}>
+        <div className={`${styles.head}`}>
+          Give wings to
+          <br />
+          your <span className={`${styles.dreams}`}>dreams</span>
+          <hr className={`${styles.hr}`} />
+          <button className={`border-0 ${styles.btn1}`}>About Us</button>
```

Comment:
This is fine `className={`border-0 ${styles.btn1}`}` when using bootstrap+custom css use this syntax